docs/Getting-Started-GPU.md

```diff
@@ -265,16 +303,15 @@ Something is wrong and this doesn't work right now on the NLP cluster. In theory
 # On the Stanford NLP cluster, you might need this:
 export PATH=$(echo $PATH | sed 's|:/usr/local/cuda/bin||')
 
-# Activate your virtual environment
-source levanter/bin/activate
-
-srun --nodes=2 python -m levanter.main.train_lm --config config/gpt2_small.yaml --trainer.per_device_parallelism -1
+CONTAINER_PATH="ghcr.io/nvidia/jax:levanter"
+TRAINING_COMMAND="python -m levanter.main.train_lm --config_path config/gpt2_7b.yaml --trainer.ray.auto_start_cluster false --trainer.per_device_parallelism -1"
 
-Submit the job with sbatch:
-
-```bash
-sbatch my-job.sh
+srun docker run --gpus=all --shm-size=16g --rm $CONTAINER_PATH $TRAINING_COMMAND
```

Comment:
Pretty sure srun + docker run is wrong. Need to test on the NLP cluster.

Reply:
huh i thought i had tested that, but i dunno